packages/react-query/src/ssg/index.ts

```diff
@@ -1,10 +1,10 @@
-import { createSSGHelpers } from './ssg';
+import { createServerSideHelpers } from './ssg';
 
 export {
-  createSSGHelpers,
+  createServerSideHelpers,
   /**
-   * @deprecated - use `createSSGHelpers` instead
+   * @deprecated - use `createServerSideHelpers` instead
    */
-  createSSGHelpers as createProxySSGHelpers,
+  createServerSideHelpers as createProxySSGHelpers,
```

Comment:
plz keep a a test that uses `createProxySSGHelpers`

Reply:
or maybe copy/paste the old unchanged files to a `./deprecated`-folder

Reply:
Ah I thought of that, but accidentally did it only in the dehydration test. I'll move both to a `./deprecated` folder, I think that sounds neatest.